do_cmake.sh

```diff
@@ -14,7 +14,7 @@ if [ -e $BUILD_DIR ]; then
 fi
 
 PYBUILD="3"
-ARGS="-GNinja"
+ARGS+="-GNinja"
```

Comment:
i cannot find any reasonings or explanations in the commit message. it reads like a bug report.

Reply:
you are right! updated, thanks! :)